docs/attributes-registry/exception.md

```diff
@@ -23,15 +25,9 @@ It is usually not possible to determine at the point where an exception is throw
 whether it will escape the scope of a span.
 However, it is trivial to know that an exception
 will escape, if one checks for an active exception just before ending the span,
-as done in the [example for recording span exceptions](#recording-an-exception).
+as done in the [example for recording span exceptions](https://opentelemetry.io/docs/specs/semconv/exceptions/exceptions-spans/#recording-an-exception).
 
 It follows that an exception may still escape the scope of the span
 even if the `exception.escaped` attribute was not set or set to false,
 since the event might have been recorded at a time where it was not
 clear whether the exception will escape.
-<!-- endsemconv -->
-
-### Recording An Exception
-
-The `exception.escaped` attribute has special semantics in the context of
-a span. Please read the [details here](../exceptions/exceptions-spans.md#recording-an-exception).
```

Comment:
We lost this now here, debatable if it's relevant or not..

Reply:
We link to it in general document now, see the update yaml